common/src/commonMain/kotlin/com/cuhacking/atlas/common/FeatureApi.kt

```diff
@@ -0,0 +1,29 @@
+package com.cuhacking.atlas.common
+
+import com.cuhacking.atlas.db.Database
+import io.github.dellisd.spatialk.geojson.Feature
+import io.github.dellisd.spatialk.geojson.FeatureCollection.Companion.toFeatureCollection
+import io.ktor.client.*
+import io.ktor.client.request.*
+
+class FeatureApi (private val database: Database, private val client: HttpClient) {
+
+    private fun Feature.toDataFeature() = com.cuhacking.atlas.db.Feature(
```

Comment:
```suggestion
    private fun Feature.toDataFeature() = DbFeature(
```
You can import this as an alias.
```kotlin
import com.cuhacking.atlas.db.Feature as DbFeature
```